packages/server/src/api/index.ts

```diff
@@ -171,25 +171,16 @@ router.get('/user/topics', async (req, res) => {
  * Get the single topic by id
  */
 router.get('/topic/:id', async (req, res) => {
-    const idSchema = z
-        .number({
-            required_error: 'id is required',
-            invalid_type_error: 'id must be a number'
-        })
-        .refine((value) => value !== 0, {
-            message: 'id must not be equal to 0',
-        });
-
+    let parsedParam;
```

Comment:
Avoid usage of `let` keyword with unassigned variable.

Reply:
Hope you understand why i am reviewing your code so critically. This is just to make sure this project is at it's best. Thanks

Reply:
Yes I understand and I really like your fast and on point feedback.